app/controllers/AccountManagement.scala

```diff
@@ -37,25 +37,25 @@ object SessionSubscription {
 
   val SUBSCRIPTION_SESSION_KEY = "subscriptionId"
 
-  def set(result: Result, sub: Subscription[SubscriptionPlan.PaperPlan]) =
+  def set(result: Result, sub: Subscription[Paid]): Result =
     result.withSession(
       SUBSCRIPTION_SESSION_KEY -> sub.name.get
     )
 
   def clear(result: Result)(implicit request: Request[AnyContent]): Result =
     result.withSession(request.session - SUBSCRIPTION_SESSION_KEY)
 
-  def subscriptionFromRequest(implicit request: Request[_]): Future[Option[Subscription[SubscriptionPlan.PaperPlan]]] = {
+  def subscriptionFromRequest(implicit request: Request[_]): Future[Option[Subscription[Paid]]] = {
```

Comment:
so now people can manage digipack and membership now
or perhaps the product should be ContentSubscripton if we only want to pull in digipack